### PR TITLE
Bump to v5.2.1

### PR DESCRIPTION
**Pull request by mheon**
As the title says

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by lsm5 — APPROVED** [review]
/approve
/lgtm
/hold

File: version/rawversion/version.go [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package rawversion
-
+65;7402;1c
```
Comment (lsm5) [inline version/rawversion/version.go:2]: @mheon whoops. some bad chars here

**Comment by mheon** [comment]
@lsm5 Oops, fixed

**Review by lsm5 — APPROVED** [review]
/approve
/lgtm
/hold

**Comment by Luap99** [comment]
```
Checking quay.io test image accessibility
    quay.io/libpod/alpine:latest
time="2024-08-14T12:44:20Z" level=fatal msg="Error parsing image name \"docker://quay.io/libpod/alpine:latest\": can't talk to a V1 container registry"
```
I think we should drop this check now given we mostly use a local registry now
cc @cevich @edsantiago

**Comment by Luap99** [comment]
@mheon cirrus.yml DEST_BRANCH is not correct likely causing the bloat warnings as it compares against the wrong branch I think.

**Comment by edsantiago** [comment]
> I think we should drop this check now given we mostly use a local registry now cc @cevich @edsantiago

Agreed. We still can't live without quay, but this check here doesn't gain us anything. Context: https://github.com/containers/podman/blob/663833745389645823acf10362a098d73f9dd024/contrib/cirrus/prebuild.sh#L112-L119

**Comment by mheon** [comment]
@Luap99 Fixed

**Review by Luap99 — APPROVED** [review]
/lgtm

**Comment by mheon** [comment]
/hold cancel

Machine flaked, but should merge once that reruns

**Comment by mheon** [comment]
Restarting the Bud test but it didn't look like a flake to me.

**Comment by Luap99** [comment]
> Restarting the Bud test but it didn't look like a flake to me.

It passed on https://github.com/containers/podman/pull/23606 so likely flake.

**Comment by mheon** [comment]
The machine test has failed 3 times in a row now. I think this isn't a flake.

**Pull request merged**